src/rpc/client.cpp

```diff
@@ -242,7 +242,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
 
     { "setgov", 0, "variables" },
     { "setgov", 1, "inputs" },
-    { "isappliedcustomtx", 0, "txid" },
+
     { "isappliedcustomtx", 1, "blockHeight" },
```

Comment:
How about the blockheight, or do we need the conversion for string to number?

Reply:
Yes，so need to keep the block height here.

Reply:
yes, (1) should stay, (0) may be killed